tagstudio/resources/translations/en.json

```
@@ -1,120 +1,163 @@
+    "entries.unlinked.refresh_all": "&Refresh All",
+    "entries.unlinked.relink.attempting": "Attempting to Relink {idx}/{missing_count} Entries, {fixed_count} Successfully Relinked",
+    "entries.unlinked.relink.manual": "&Manual Relink",
     "entries.unlinked.relink.title": "Relinking Entries",
     "entries.unlinked.scanning": "Scanning Library for Unlinked Entries...",
-    "entries.unlinked.search_and_relink": "Search && Relink",
+    "entries.unlinked.search_and_relink": "&Search && Relink",
     "entries.unlinked.title": "Fix Unlinked Entries",
+    "entries.unlinked.missing_count.none": "Unlinked Entries: N/A",
+    "entries.unlinked.missing_count.some": "Unlinked Entries: {count}",
     "field.copy": "Copy Field",
     "field.edit": "Edit Field",
     "field.paste": "Paste Field",
     "file.date_added": "Date Added",
     "file.date_created": "Date Created",
     "file.date_modified": "Date Modified",
     "file.dimensions": "Dimensions",
+    "file.duplicates.description": "TagStudio supports importing DupeGuru results to manage duplicate files.",
     "file.duplicates.dupeguru.advice": "After mirroring, you're free to use DupeGuru to delete the unwanted files. Afterwards, use TagStudio's \"Fix Unlinked Entries\" feature in the Tools menu in order to delete the unlinked Entries.",
     "file.duplicates.dupeguru.file_extension": "DupeGuru Files (*.dupeguru)",
-    "file.duplicates.dupeguru.load_file": "Load DupeGuru File",
+    "file.duplicates.dupeguru.load_file": "&Load DupeGuru File",
     "file.duplicates.dupeguru.no_file": "No DupeGuru File Selected",
     "file.duplicates.dupeguru.open_file": "Open DupeGuru Results File",
     "file.duplicates.fix": "Fix Duplicate Files",
     "file.duplicates.matches_uninitialized": "Duplicate File Matches: N/A",
-    "file.duplicates.matches": "Duplicate File Matches: %{count}",
-    "file.duplicates.mirror_entries": "Mirror Entries",
+    "file.duplicates.matches": "Duplicate File Matches: {count}",
+    "file.duplicates.mirror_entries": "&Mirror Entries",
     "file.duplicates.mirror.description": "Mirror the Entry data across each duplicate match set, combining all data while not removing or duplicating fields. This operation will not delete any files or data.",
     "file.duration": "Length",
     "file.not_found": "File Not Found",
     "file.open_file_with": "Open file with",
     "file.open_file": "Open file",
-    "file.open_location.generic": "Show file in explorer",
+    "file.open_location.generic": "Show file in file explorer",
     "file.open_location.mac": "Reveal in Finder",
-    "file.open_location.windows": "Show in File Explorer",
+    "file.open_location.windows": "Show in Explorer",
```

Recommend reverting back to ["File Explorer"](https://support.microsoft.com/en-us/windows/windows-explorer-has-a-new-name-c95f0e92-b1aa-76da-b994-36a7c7c413d7) for Windows
```suggestion
    "file.open_location.windows": "Show in File Explorer",
```